content/id/docs/contribute/suggesting-improvements.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Menyarankan peningkatan konten
+slug: suggest-improvements
+content_type: concept
+weight: 10
+card:
+  name: contribute
+  weight: 20
+---
+
+<!-- overview -->
+
+Jika kamu menemukan masalah pada dokumentasi Kubernetes, atau mempunyai ide untuk
+konten baru, maka silakan untuk membuat isu pada Github. Kamu hanya membutuhkan
```

Comment:
oh I learned that "silakan" is the formal one instead of "silahkan" https://kbbi.kemdikbud.go.id/entri/silakan